.github/workflows/run-tests.yml

```diff
@@ -19,14 +20,14 @@ jobs:
       # but notably it does NOT override the default PROJECT_VERSION, so the
       # farmOS Composer project 2.x branch is always used.
       - name: Build farmOS 2.x-dev Docker image
-        run: docker build --build-arg FARMOS_REPO=https://github.com/${GITHUB_REPOSITORY} --build-arg FARMOS_VERSION=${FARMOS_VERSION} -t farmos/farmos:2.x-dev docker/dev
+        run: docker build --build-arg FARMOS_REPO=https://github.com/${GITHUB_REPOSITORY} --build-arg FARMOS_VERSION=${FARMOS_VERSION} --build-arg WWW_DATA_ID=33 -t farmos/farmos:2.x-dev docker/dev
       - name: Create docker-compose.yml
-        run: curl https://raw.githubusercontent.com/farmOS/farmOS/2.x/docker/docker-compose.development.yml -o docker-compose.yml
+        run: cp docker/docker-compose.testing.yml docker-compose.yml
       - name: Start containers
-        run: sudo docker-compose up -d
-      - name: Wait 10 seconds
-        run: sleep 10
+        run: docker-compose up -d
+      - name: Wait 15 seconds
+        run: sleep 15
```

Comment:
Increased this to 15 seconds to allow for the previous 10 seconds plus startup of the test-runner container.